docs/rules/require-hook.md

```diff
@@ -148,3 +148,74 @@ afterEach(() => {
   clearCityDatabase();
 });
 ```
+
+## Options
+
+Some test utils provides methods which takes hook as an argument 
+and should be executed outside a hook.
+
+For example https://vue-test-utils.vuejs.org/api/#enableautodestroy-hook 
+which takes the hook as an argument. To exclude them you can update settings
+
+```json
+{
+  "jest/require-hook": [
+    "error",
+    {
+      "excludedFunctions": [
+        "enableAutoDestroy"
+      ]
+    }
+  ]
+}
+```
+
+Examples of **incorrect** code for the `{ "excludedFunctions": ["enableAutoDestroy"] }`
+option:
+
+```js
+/* eslint jest/require-hook: ["error", { "excludedFunctions": ["enableAutoDestroy"] }] */
+
+import {
+  enableAutoDestroy,
+  resetAutoDestroyState,
+  mount
+} from '@vue/test-utils';
+import initDatabase from './initDatabase';
+
+enableAutoDestroy(afterEach);
+initDatabase(); // this will throw a linting error
+
+describe('Foo', () => {
+  test('always returns 42', () => {
+    expect(global.getAnswer()).toBe(42);
+  })
+})
+```
+
+
+Examples of **correct** code for the `{ "excludedFunctions": ["enableAutoDestroy"] }`
+option:
+
+```js
+/* eslint jest/require-hook: ["error", { "excludedFunctions": ["enableAutoDestroy"] }] */
+
+import {
+  enableAutoDestroy,
+  resetAutoDestroyState,
+  mount
+} from '@vue/test-utils';
+import {initDatabase, tearDownDatabase} from './databaseUtils';
```

Comment:
Personally I think it's good to try and keep the "correct" and "incorrect" examples similar, so think we should either stick with just having `initDatabase` or have `tearDownDatabase` in both examples.